packages/client/auth/src/CrossmintAuthClient.ts

```diff
@@ -232,7 +232,10 @@ export class CrossmintAuthClient extends CrossmintAuth {
 
     public async authenticateSmartWallet(address: string, signature: string) {
         try {
-            const queryParams = new URLSearchParams({ signinAuthenticationMethod: "evm" });
+            const queryParams = new URLSearchParams({
+                signinAuthenticationMethod: "evm",
+                callbackUrl: `${this.apiClient.baseUrl}/${AUTH_SDK_ROOT_ENDPOINT}/callback`,
```

Comment:
we should remove it from the backend then and remove it here too. that was part of the point of the refactor and removing dead code. could you do quick PRs?

Reply:
I've already got the backend PR in progress (just waiting on client migration)